src/pynwb/core.py

```diff
@@ -80,6 +80,21 @@ def __init__(self, **kwargs):
         call_docval_func(super(NWBBaseType, self).__init__, kwargs)
         self.__fields = dict()
 
+    def get_parent_neurodata_type(self, neurodata_type='NWBFile'):
+        """
+        Traverse parent hierarchy and return first instance of the
+        specified neurodata_type
+
+        Args:
+            neurodata_type (str)    : the neurodata_type to search for. Search for *NWBFile* by default
+        """
+        p = self.parent
+        while p is not None:
+            if p.neurodata_type == neurodata_type:
+                return p
+            p = p.parent
+        return p
```

Comment:
@ajtritt Can we just do `return`  here? This is easier to grasp IMHO. Or leave it out entirely.

Reply:
Good catch. `return None` seems even easier to grasp. Do you object?